systems/plants/collision/DrakeCollision.h

```diff
@@ -9,171 +9,26 @@
-				   const int body_collision_idx, 
-				   Eigen::Vector3d &ptA, Eigen::Vector3d &ptB, 
-				   Eigen::Vector3d &normal) { return false; };
-
-    virtual bool getPairwiseCollision(const int bodyA_idx, const int bodyB_idx, 
-				      Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB, 
-				      Eigen::MatrixXd& normals) { return false; };
-
-    virtual bool getPairwisePointCollision(const int bodyA_idx, const int bodyB_idx, 
-					   const int body_collisionA_idx, 
-					   Eigen::Vector3d &ptA, Eigen::Vector3d &ptB, 
-					   Eigen::Vector3d &normal) { return false; };
-
-    virtual bool getClosestPoints(const int bodyA_idx, const int bodyB_idx,
-				  Eigen::Vector3d& ptA, Eigen::Vector3d& ptB, Eigen::Vector3d& normal,
-				  double& distance) { return false; };
-
-    virtual bool closestPointsAllBodies(std::vector<int>& bodyA_idx, 
-					std::vector<int>& bodyB_idx, 
-					Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB,
-					Eigen::MatrixXd& normal, 
-					Eigen::VectorXd& distance,
-					const std::vector<int>& bodies_idx,
-          const std::set<std::string>& active_element_groups) { return false; };
-
-    bool closestPointsAllBodies(std::vector<int>& bodyA_idx, 
-        std::vector<int>& bodyB_idx, 
-        Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB,
-        Eigen::MatrixXd& normal, 
-        Eigen::VectorXd& distance,
-        const std::set<std::string>& active_element_groups);
-
-    bool closestPointsAllBodies(std::vector<int>& bodyA_idx, 
-        std::vector<int>& bodyB_idx, 
-        Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB,
-        Eigen::MatrixXd& normal, 
-        Eigen::VectorXd& distance,
-        const std::vector<int>& bodies_idx);
 
-    bool closestPointsAllBodies(std::vector<int>& bodyA_idx, 
-        std::vector<int>& bodyB_idx, 
-        Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB,
-        Eigen::MatrixXd& normal, 
-        Eigen::VectorXd& distance);
+  DLLEXPORT_drakeCollision std::unique_ptr<Model> newModel();
 
-    virtual bool allCollisions(std::vector<int>& bodyA_idx, 
-			       std::vector<int>& bodyB_idx, 
-			       Eigen::MatrixXd& ptsA, Eigen::MatrixXd& ptsB) { return false; };
-                                  
-    //
-    // Performs raycasting collision detecting (like a LIDAR / laser rangefinder)
-    //
-    // @param origin Vector3d specifying the position of the ray's origin
-    // @param ray_endpoint Vector3d specifying a second point on the ray in world coordinates
-    // @param distance to the first collision, or -1 on no collision
-    //
-    virtual bool collisionRaycast(const Eigen::Matrix3Xd &origin, const Eigen::Matrix3Xd &ray_endpoint, Eigen::VectorXd &distances) { return false; };
+  typedef std::bitset<MAX_NUM_COLLISION_FILTER_GROUPS> bitmask;
 
-    protected:
-      virtual const std::vector<int> bodyIndices() const;
-      virtual const std::set<std::string> elementGroupNames() const;
-  };
-
-  DLLEXPORT std::shared_ptr<Model> newModel();
```

Comment:
On second look, this is fine - those lines with the wrong DLLEXPORT are being removed by the diff, not added.